### ISSUE TITLE
improve accessibility and disabled user access

### ISSUE THREAD

**Issue by darcher-figo** [darcher-figo/figo-rts#13]
ALL `svg` tags should receive `focusable="true"` and `role="img"` to prevent screen readers from going into the vector frame, but there's a lot of areas that require some accessibility improvements, we need to investigate and isolate these areas, and create backlog tickets to go after them before they cause any legality issues.

`aria-describedby`, `aria-controlledby`, `aria-live="polite"`, `aria-label`, etc. etc. 

For example, one approach:

For form validation, as fields become "valid" or "populated" the progression button should be disabled and have an `aria-describedby` with a comma delimited list of field `id`'s for fields that remain prior to the button being enabled.

or

For modals, `aria-hidden="true"` should be applied to the wrapper of all page content behind the modal, while the modal, outside of the container with this attribute, should have proper `dialog`/`modal` `roles` and `aria-live` indicators on close buttons, proceed buttons, etc.
